usaspending_api/etl/helpers.py

```diff
@@ -1,9 +1,18 @@
 from datetime import datetime
+import warnings
 
 from django.db.models import Q
+from django.core.cache import caches, CacheKeyWarning
 
 from usaspending_api.references.models import Agency, Location, RefCountryCode
 
+warnings.simplefilter("ignore", CacheKeyWarning)
+
+
+def clear_caches():
+    for cache in ['locations', 'recipients', 'summary-awards']:
```

Comment:
good point!  Taking out that cache anyway for now, but if we ever re-do we can use a better name